docs/basics/101-180-FAQ.rst

```diff
@@ -411,6 +411,21 @@ dataset.
 When using the terminal command ``cp`` [#f1]_, it is sufficient to use the
 ``-L``/``--dereference`` option. This will follow symbolic links, and make
 sure that content gets moved instead of symlinks.
+Remember that if you are copying some annexed content out of a dataset without
+unlocking it first, you will only have "read" access on the files you have just 
+copied. Therefore you can :
+- either unlock the files before copying them out, 
```

Comment:
Note to self: write a glossary entry about "unlocking"

Reply:
Want me to add a stub with a TODO in the glossary as part of this PR?